src/screens/EventVolunteers/VolunteerGroups/VolunteerGroups.tsx

```
@@ -16,31 +16,29 @@
  *
  * @returns The rendered volunteer groups management component.
  */
-import React, { useCallback, useMemo, useState, useEffect } from 'react';
+import React, { useMemo, useState, useEffect } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Button } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)